tests/team.test.js

```diff
@@ -354,6 +354,102 @@ describe("PATCH change team", function () {
                     res.body.message.should.equal(Constants.Success.TEAM_JOIN);
                     res.body.should.have.property("data");
 
+                    done();
+                });
+        });
+    });
+});
+
+describe("PATCH change team info", function () {
+    it("should FAIL to change a hacker's team information", function (done) {
+        chai.request(server.app)
+            .patch(`/api/team/${util.hacker.HackerF._id}`)
+            .type("application/json")
+            .send({
+                name: "BronzeTeamASDF",
+            })
+            .end(function (err, res) {
+                res.should.have.status(401);
+                res.should.be.json;
+                res.body.should.have.property("message");
+                res.body.message.should.equal(Constants.Error.AUTH_401_MESSAGE);
+                res.body.should.have.property("data");
+
+                done();
+            });
+    });
+
+    it("should FAIL for a hacker to change another team's information", function (done) {
```

Comment:
Describe failure case: "should FAIL to change a hacker's team information due to invalid authorization"